roadmap/1_core-protocol/upcoming/1_canister-groups/canister-groups.md

```diff
@@ -0,0 +1,13 @@
+-
+title: Canister groups
+links:
+  Forum Link: https://forum.dfinity.org/t/canister-groups/16015
+  Proposal: 
+eta:
+is_community: false
+---
+
+Communication between canisters on the same subnetwork is very fast since the messages that they exchange do not need to go through consensus. Since intercanister communication latency is less of a concern, one can scale Dapps through multi-canister architectures, as long as the canisters are guaranteed to be collocated. 
+
+Since upcoming load-balancing mechanisms need to move canisters between subnets, we need to avoid splitting up canisters that are part of the same Dapp.   This feature would allow a Dapp developer to explicitly indicate which canisters "belong" together and which should always be located on the same subnet.
```

Comment:
```suggestion
Since upcoming load-balancing mechanisms need to move canisters between subnets, we need to avoid splitting up canisters that are part of the same dapp. This feature would allow a dapp developer to explicitly indicate which canisters "belong" together and which should always be located on the same subnet.
```

Reply:
I'd probably rephrase the last sentence as "Canister groups will [...]" – since it's on the roadmap so I would expect it to be built.